nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/WEB-INF/partials/canvas/policy-management.jsp

```diff
@@ -44,16 +44,6 @@
                 </div>
                 <div class="clear"></div>
             </div>
-            <div id="policy-selected-template-container" class="hidden policy-selected-component-container">
-                <div class="policy-selected-component-type-icon">
-                    <i class="icon icon-template"></i>
-                </div>
-                <div class="policy-selected-component-details-container">
-                    <div class="policy-selected-component-name"></div>
-                    <div class="policy-selected-component-type">Template</div>
-                </div>
-                <div class="clear"></div>
-            </div>
```

Comment:
I don't believe that this policy template has anything to do with flow templates. We should put this back.

Reply:
hmm, maybe now that i think about it a bit more... maybe they are the same.

@scottyaslan do you know?